### PR TITLE
Fixed a bug - 🎏Glitch

### PR DESCRIPTION
**Pull request by callumok2004**

### CODE REVIEW

**Comment by TheDropX** [comment]
Please upload an screenshot too.

**Comment by callumok2004** [comment]
https://downbeat.is-inside.me/OuvHLE4E.png

**Comment by callumok2004** [comment]
OuvHLE4E 1

**Comment by callumok2004** [comment]
Just updated another presence to, fixing the service name, not being the same as the folder name, so the source code button led to a 404 page

File: FnBr/dist/metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "name": "Not Officer",
         "id": "373913699943186432"
     },
-    "service": "fnbr.co",
+    "service": "FnBr",
```
Comment (Bas950) [inline FnBr/dist/metadata.json]: fnbr.co was correct, folder name needs changing

**Pull request merged**